darts/dataprocessing/transformers/boxcox.py

```diff
@@ -88,15 +91,37 @@ def _transform_iterator(self, series: Sequence[TimeSeries]) -> Iterator[Tuple]:
     def _inverse_transform_iterator(self, series: Sequence[TimeSeries]) -> Iterator[Tuple]:
         return zip(series, self._fitted_params)
 
+    @staticmethod
+    def _reshape_in(series: TimeSeries) -> np.ndarray:
```

Comment:
I imagine that methods `_reshape_in/_out` can be useful for all transformers (as redefined in Scaler). Could these be part of `BaseDataTransformer`?

Reply:
Indeed, absolutely!